bootloader/bootloadertest/bootloadertest.go

```diff
@@ -97,3 +97,13 @@ func (b *MockBootloader) SetBootKernel(kernel string) {
 func (b *MockBootloader) SetBootBase(base string) {
 	b.SetBootVars(map[string]string{"snap_core": base})
 }
+
+func (b *MockBootloader) SetTrying() {
```

Comment:
this should probably have Reboot in the name (also for symmetry with the next one),  SetTryingDuringReboot ?